client/context.go

```diff
@@ -25,7 +25,7 @@ type Context struct {
 	FromAddress       sdk.AccAddress
 	Client            rpcclient.Client
 	ChainID           string
-	JSONMarshaler     codec.JSONMarshaler
```

Comment:
Was there a reason to not update the context field as well?

Reply:
#9251